packages/office-ui-fabric-react/src/components/TextField/examples/TextField.CustomRender.Example.tsx

```diff
@@ -37,8 +39,19 @@ export class TextFieldCustomRenderExample extends React.Component<
           <IconButton iconProps={{ iconName: 'Info' }} title="Info" ariaLabel="Info" onClick={this._onClick} />
         </span>
         {isCalloutVisible && (
-          <Callout className="ms-CustomRenderExample-callout" target={this._iconButtonElement} onDismiss={this._onDismiss}>
-            <text> In additon to the label itself, this label includes an iconbutton which pops out more information in a callout</text>
+          <Callout
+            className="ms-CustomRenderExample-callout"
+            target={this._iconButtonElement}
+            setInitialFocus={true}
+            onDismiss={this._onDismiss}
+            ariaDescribedBy={this._descriptionId}
+          >
+            <div className="ms-CustomRenderExample-callout-description" id={this._descriptionId}>
```

Comment:
Can you use either inline styles or mergeStyles instead, and get rid of the extra wrapping divs?